app/serializers/response_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class ResponseSerializer
+  include JSONAPI::Serializer
+  attributes :id, :description, :date, :price
+  # belongs_to :topic
+  # belongs_to :user
```

Comment:
Same here `topic` and `user` should be serialized since those are associations to Response.